src/core/simulation.jl

```diff
@@ -223,22 +218,34 @@ end
 """
 mutable struct Simulation
     steps::Int
-    stages::Dict{String, OperationsProblem{<:AbstractOperationsProblem}}
+    problems::SimulationProblems
```

Comment:
```
Simulation
    SimulationSequence
    SimulationProblems
        Dict{OperationsProblem}
            ProblemInternal
                SimulationInfo
```
`SimulationInfo` doesn't seem quite right to me.

Reply:
In what sense?

Reply:
How about ProblemSimulationData that could get closer to SimulationStoreProblemParams ?